### ISSUE TITLE
[false positive] Yoda conditions for ranges

### ISSUE THREAD

**Issue by HairyFotr** [HairyFotr/linter#10]
When you have a range of values, this is the preffered style, and Linter shouldn't warn:

```
if (0 < x && x < 10) ...
```
